locales/fr_FR/pcsx2_Main.po

```diff
@@ -2892,11 +2883,11 @@ msgstr ""
 
 #: pcsx2/gui/Panels/VideoPanel.cpp:209
 msgid "Frames to Draw"
-msgstr "Frames à rendre"
+msgstr "Images à Dessiner"
```

Comment:
I don't speak french but is this the right type or "Draw"? Since the draw we want is in relation to rendering to the picture.

Reply:
Hi, 

Draw is not meaning like "draw a sketch" ? So I'll change it

Reply:
I would say "render" is kind of closer than drawing, although they are technically to the same result (aka a picture) I guess it's just how that's achieved is different? IDK, as I said, I'm not french so IDK if you need to be more specific like that in French or not, I just don't want people getting pissy because we have the wrong type of drawing :P

Reply:
Could you change by "Images à rendre" ?

Sometimes it's difficult to translate some english terms. It's good to have a context :) Thanks for your work on PCSX2.

Reply:
"Images à rendre", or maybe "Trames à rendre" seems more appropriate indeed.